packages/venia-concept/src/components/OnlineIndicator/onlineIndicator.css

```diff
@@ -1,67 +1,8 @@
 .root {
     align-items: center;
-    display: inline-flex;
-    font-size: 0.9em;
-    height: 0px;
-    justify-content: flex-start;
-    margin-bottom: 0.3em;
-    overflow: hidden;
-    padding: 0 2.4rem;
-    position: sticky;
-    top: 0;
-    width: 100%;
-}
-
-.offline {
-    composes: root;
-    background-color: rgb(var(--venia-warning-light));
-    animation-name: slidein;
-    animation-duration: 244ms;
-    animation-fill-mode: forwards;
-}
-
-.online {
-    composes: root;
-    background-color: rgb(var(--venia-teal-light));
-    animation-name: slideinandhide;
-    animation-duration: 3s;
-    animation-fill-mode: forwards;
-}
-
-.offline span {
-    color: rgb(var(--venia-warning-dark));
-}
-
-.online span {
-    color: rgb(var(--venia-teal));
-}
-
-.root p {
-    margin-left: 1em;
-    line-height: 1.3em;
-}
-
-@keyframes slidein {
-    from {
-        height: 0;
-    }
-    to {
-        height: 3rem;
-    }
-}
-
-@keyframes slideinandhide {
-    0% {
-        height: 3rem;
-        opacity: 0.5;
-    }
-    10% {
-        opacity: 1;
-    }
-    80% {
-        height: 3rem;
-    }
-    100% {
-        height: 0;
-    }
+    display: flex;
+    grid-area: primary;
+    height: 3rem;
+    margin-left: 4.5rem;
+    width: 3rem;
```

Comment:
Is this no longer going to be a custom indicator? Are we just making it a toast now?

Reply:
The indicator still exists though it only displays when offline and it now appears as an icon in the header to the left of the logo. Try offline mode and you'll see.